modules/apps/frontend-editor/frontend-editor-ckeditor-web/src/main/resources/META-INF/resources/resources.jsp

```diff
@@ -35,6 +35,34 @@ String inlineEditSaveURL = GetterUtil.getString((String)request.getAttribute(CKE
 	long javaScriptLastModified = PortalWebResourcesUtil.getLastModified(PortalWebResourceConstants.RESOURCE_TYPE_EDITOR_CKEDITOR);
 	%>
 
+	<script data-senna-track="temporary" type="text/javascript">
+		window.CKEDITOR_GETURL = function (resource) {
+			var languageParameter = '&languageId=' + themeDisplay.getLanguageId();
+
+			// If this is not a full or absolute path.
```

Comment:
I suspect Brian won't want a trivial comment like this in here; so if you remove it, you'll increase your chances of getting it merged first time.

Reply:
Yes, I'll remove that